test/e2e/tests/state-snapshots/sentry-mask-policy-gaps.json

```diff
@@ -0,0 +1,130 @@
+  "announcements.10.image.width",
+  "announcements.10.isShown",
+  "announcements.11.date",
+  "announcements.11.id",
+  "announcements.11.isShown",
+  "announcements.12.date",
+  "announcements.12.id",
+  "announcements.12.image.src",
+  "announcements.12.image.width",
+  "announcements.12.isShown",
+  "announcements.13.date",
+  "announcements.13.id",
+  "announcements.13.isShown",
+  "announcements.14.date",
+  "announcements.14.id",
+  "announcements.14.isShown",
+  "announcements.15.date",
+  "announcements.15.id",
+  "announcements.15.isShown",
+  "announcements.16.date",
+  "announcements.16.id",
+  "announcements.16.isShown",
+  "announcements.17.date",
+  "announcements.17.id",
+  "announcements.17.isShown",
+  "announcements.18.date",
+  "announcements.18.id",
+  "announcements.18.isShown",
+  "announcements.19.date",
+  "announcements.19.id",
+  "announcements.19.isShown",
+  "announcements.20.date",
+  "announcements.20.id",
+  "announcements.20.isShown",
+  "announcements.21.date",
+  "announcements.21.id",
+  "announcements.21.isShown",
+  "announcements.22.date",
+  "announcements.22.id",
+  "announcements.22.isShown",
+  "announcements.23.date",
+  "announcements.23.id",
+  "announcements.23.isShown",
+  "browserEnvironment.browser",
+  "browserEnvironment.os",
+  "customNonceValue",
+  "database",
+  "disabledRpcMethodPreferences.eth_sign",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.address",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.lastSelected",
+  "identities.0x5cfe73b6021e818b776b421b1c4db2474086a7e1.name",
+  "isAccountMenuOpen",
+  "isInitialized",
+  "isNetworkMenuOpen",
+  "keyringTypes.0",
+  "keyringTypes.1",
+  "keyringTypes.2",
+  "keyringTypes.3",
+  "keyringTypes.4",
+  "keyringTypes.5",
+  "lastUpdated",
+  "networkConfigurations.networkConfigurationId.chainId",
+  "networkConfigurations.networkConfigurationId.id",
+  "networkConfigurations.networkConfigurationId.networkConfigurationId",
+  "networkConfigurations.networkConfigurationId.nickname",
+  "networkConfigurations.networkConfigurationId.rpcUrl",
+  "networkConfigurations.networkConfigurationId.ticker",
+  "networksMetadata.networkConfigurationId.EIPS.1559",
+  "networksMetadata.networkConfigurationId.status",
+  "nextNonce",
+  "pendingApprovalCount",
+  "previousUserTraits.address_book_entries",
+  "previousUserTraits.install_date_ext",
+  "previousUserTraits.ledger_connection_type",
+  "previousUserTraits.networks_added.0",
+  "previousUserTraits.nft_autodetection_enabled",
+  "previousUserTraits.number_of_accounts",
+  "previousUserTraits.number_of_nft_collections",
+  "previousUserTraits.number_of_nfts",
+  "previousUserTraits.number_of_tokens",
+  "previousUserTraits.theme",
+  "previousUserTraits.three_box_enabled",
+  "previousUserTraits.token_detection_enabled",
+  "previousUserTraits.undefined",
+  "smartTransactionsState.liveness",
+  "snapsInstallPrivacyWarningShown",
+  "termsOfUseLastAgreed",
```

Comment:
Seems to be missing from the app-state controller

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b9210b7553f2db3b7dbb53c3da9bf7f71cf7ee14